Mathlib/Algebra/Algebra/Subalgebra/Basic.lean

```diff
@@ -637,6 +641,10 @@ instance : CompleteLattice (Subalgebra R A) where
   bot := (Algebra.ofId R A).range
   bot_le _S := fun _a ⟨_r, hr⟩ => hr ▸ algebraMap_mem _ _
 
+theorem sup_def (S T : Subalgebra R A) : S ⊔ T = adjoin R (S ∪ T : Set A) := rfl
+
+theorem sSup_def (S : Set (Subalgebra R A)) : sSup S = adjoin R (⋃₀ (SetLike.coe '' S)) := rfl
```

Comment:
The rhs here looks like it should be stated with iSup

Reply:
I'm not sure about this. Since LHS is `s` version, I think the RHS should also be `s` version, i.e. `sUnion`. Could you give an equivalent statement in `iUnion`?